raiden/transfer/token_network.py

```diff
@@ -30,50 +30,88 @@
 ]
 
 
-def subdispatch_to_channel_by_id_and_address(
+def subdispatch_to_channels_by_participant_address(
     token_network_state: TokenNetworkState,
     state_change: StateChangeWithChannelID,
     block_number: BlockNumber,
     block_hash: BlockHash,
-    node_address: AddressHex = None
+    participant_address: AddressHex = None
 ) -> TransitionResult:
-    events = list()
+    if not participant_address:
```

Comment:
it seems a bit strange to me that a caller would omit the `participant_address` parameter if this method is just going to return an empty `TransitionResult`. but i guess this is out of scope for this PR